enumfields/fields.py

```diff
@@ -34,14 +34,29 @@ def __set__(self, obj, value):
         obj.__dict__[self.field.name] = python_value
 
 
+class CastOnSaveDescriptor:
+
+    def __get__(self, obj, type=None):
+        def save_fnc(update_fields=None, **kwargs):
+            getattr(super(obj.__class__, obj), 'save')(update_fields=update_fields, **kwargs)
```

Comment:
spíš bys to měl udělat tak, že přejmenuješ původní save metodu a tady jí budeš provolávat, Tak se obvykle monky patch dělá

Reply:
right. this has been deleted